requirements.txt

```diff
@@ -1,4 +1,3 @@
-python-dotenv==1.0.1
```

Comment:
why was this deleted?

Reply:
Huh I didn't notice that. I refroze the requirements when I added textblob so I guess if I didn't have this installed it deleted it. I'll re add it